docs/en/observability/aggregation-options.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[aggregation-options]]
+= Aggregation options
+
+Aggregations summarize your data to make it easier to analyze.
+In some alerting rules, you can specify aggregations to gather data for the rule.
+
+The following aggregations are available in some rules:
+
+|===
+|Aggregation | Description
+
+| Average
+| Average value of a numeric field.
+
+| Cardinality
+| Approximate number of unique values in a field.
+
+| Document count
+| Total number of documents in a field.
+
+| Max
+| Highest value of a numeric field.
+
+| Min
+| Lowest value of a numeric field.
+
+| Percentile
+| The point at which a specific percentage of observed values occur.
+You can choose the 95th or 99th percentile.
```

Comment:
Maybe something like this:

nth Percentile
Numeric value which represents the point at which n% of all values in the selected dataset are lower (choices are 95th or 99th)